aws/resource_aws_lex_slot_type_test.go

```diff
@@ -0,0 +1,348 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lexmodelbuildingservice"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccLexSlotType_basic(t *testing.T) {
+	resourceName := "aws_lex_slot_type.test"
+	testID := acctest.RandStringFromCharSet(8, acctest.CharSetAlpha)
+	testSlotTypeID := "test_slot_type_" + testID
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsLexSlotTypeNotExists(testSlotTypeID, "$LATEST"),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccAwsLexSlotTypeBasicConfig, testSlotTypeID),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAwsLexSlotTypeExists(testSlotTypeID, "$LATEST"),
+					resource.TestCheckResourceAttr(resourceName, "description", ""),
+					resource.TestCheckResourceAttr(resourceName, "enumeration_value.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "name", testSlotTypeID),
+					resource.TestCheckResourceAttr(resourceName, "value_selection_strategy", lexmodelbuildingservice.SlotValueSelectionStrategyOriginalValue),
+					resource.TestCheckResourceAttrSet(resourceName, "checksum"),
+					resource.TestCheckResourceAttrSet(resourceName, "version"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAwsLexSlotType_CreateVersion(t *testing.T) {
+	resourceName := "aws_lex_slot_type.test"
+	testID := acctest.RandStringFromCharSet(8, acctest.CharSetAlpha)
+	testSlotTypeID := "test_slot_type_" + testID
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsLexSlotTypeNotExists(testSlotTypeID, "$LATEST"),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccAwsLexSlotTypeCreateVersionConfig, testSlotTypeID),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAwsLexSlotTypeExists(testSlotTypeID, "$LATEST"),
+					testAccCheckAwsLexSlotTypeNotExists(testSlotTypeID, "1"),
+				),
+			},
+			{
+				ResourceName: resourceName,
+				ImportState:  true,
+				// Cannot verify import for create_version other than true because we don't have
```

Comment:
The field you are looking for here is `ImportStateVerifyIgnore` which allows you to specify which resource arguments can be ignored in the verification step since they are not part of the API response. 

```suggestion
ImportStateVerifyIgnore: []string{"create_version"}
```